client/assets/scss/colors.scss

```diff
@@ -1,4 +1,59 @@
-$blue: rgba(52, 88, 170, 1);
-$grey: rgba(106, 106, 106, 1);
-$red: rgba(204, 0, 0, 1);
-$white: rgba(255, 255, 255, 1);
\ No newline at end of file
+//blue
+$blue-whale: rgba(15, 47, 63);
+$blue-whale-75: rgba(15, 47, 63, .75);
+$blue-whale-50: rgba(15, 47, 63, .5);
+$blue-whale-25: rgba(15, 47, 63, .25);
+
+$chathams-blue: rgba(48, 87, 110);
```

Comment:
What if we're using a blue that's custom and doesn't have a name?

Reply:
they actually all colors have names. you just need to google it, i can always add a link there as a comment to tell whoever if there is a custom color. just input the rgb code and it'll give you a color name.